leakcanary-android-core/src/main/java/leakcanary/OnRetainInstanceListener.kt

```diff
@@ -0,0 +1,20 @@
+package leakcanary
+
+sealed class RetainInstanceChange {
+  data class CountChanged(val retainedCount: Int) : RetainInstanceChange()
+  object Reset : RetainInstanceChange()
```

Comment:
Oh I see this is starting to look like the API I want, but I think it should be for all types of notifications not just "retain instance change"

Reply:
Yup, will keep it private while it's baking :)